virttest/qemu_vm.py

```diff
@@ -4162,6 +4170,10 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
                     extra_params = clone.params.get("extra_params", "") + " -S"
                     clone.params["extra_params"] = extra_params
 
+                # "preconfig" is meaningless for dest, remove it whatever the value is
+                if "qemu_preconfig" in clone.params.keys():
```

Comment:
Would you please elaborate on why it is handled here instead of the place in front of the try block?

Reply:
Good catch~  only delete `qemu_preconfig` in one branch is not correct, i missed L4387, will update soon. thanks!